src/Domain/ComposerLoader.php

```diff
@@ -24,7 +24,7 @@ final class ComposerLoader
 
     public static function getInstance(Collector $collector): Composer
     {
-        if (null === self::$composer || $collector !== self::$currentCollector) {
+        if (self::$composer === null || $collector !== self::$currentCollector) {
```

Comment:
Why this change :sweat_smile: You don't like Yoda ? :wink:

Reply:
hahaaa I have a thing for Yoda style :laughing: Just kidding. I was just trying to get more scores in architecture for insight to pass the tests! So I changed them.